ako-operator/helm/ako-operator/templates/secret.yaml

```diff
@@ -7,6 +7,9 @@ type: Opaque
 data:
   username: {{ .Values.avicredentials.username | b64enc }}
   password: {{ .Values.avicredentials.password | b64enc }}
+  {{ if .Values.avicredentials.authtoken  }}
+  authtoken: {{ .Values.avicredentials.authtoken | b64enc }}
+  {{ end }}
```

Comment:
need to make the password optional too as user will provide with only 1 of the two fields

Reply:
Thanks. Addressed.